stairctl/build.gradle

```diff
@@ -19,7 +19,11 @@ dependencies {
     implementation group: 'org.apache.commons', name: 'commons-lang3'
 
     // Spring
-    implementation group: 'org.springframework.boot', name: 'spring-boot-starter'
+    implementation group: 'org.springframework.boot', name: 'spring-boot'
     // Spring Boot and Spring Shell versions should stay in sync
-    implementation group: 'org.springframework.shell', name: 'spring-shell-starter', version: '3.3.1'
+    implementation('org.springframework.shell:spring-shell-starter:3.3.1') {
+        exclude module: 'spring-boot-starter-logging'
+    }
+
+    implementation 'org.slf4j:slf4j-simple:1.7.36'
```

Comment:
This isn't necessary but it cleans up the log configuration errors you get on startup

Reply:
Fair enough, would you mind adding a comment to that effect on the exclusion?